source/install/install-rhel-8-mysql.rst

```diff
@@ -5,49 +5,66 @@ Installing MySQL Database Server
 
 Install and set up the database for use by the Mattermost server. You can install either MySQL or PostgreSQL.
 
-**To install MySQL on RHEL 8:**
+**To install MySQL on RHEL 8**
 
 1. Log in to the server that will host the database, and open a terminal window.
 
 2. Install MySQL.
 
-  ``sudo yum install mysql-community-release``
+.. note::
+ 
+  If you have additional questions for the process reference the MySQL docs `here <https://dev.mysql.com/doc/mysql-repo-excerpt/5.6/en/linux-installation-yum-repo.html>`_ that matches your system. If your RHEL is a fresh install, you'll need to add the `MySQL Yum repository <https://dev.mysql.com/doc/mysql-repo-excerpt/5.6/en/linux-installation-yum-repo.html>`_. You can also use the MySQL repo - https://repo.mysql.com/.
 
-3. Start the MySQL server.
+3. Download the latest release package using wget:
+
+  ``wget https://dev.mysql.com/get/mysql80-community-release-el8-1.noarch.rpm``
+        
+4. Once downloaded, install it: ``sudo yum localinstall platform-and-version-specific-package-name.rpm``. This will be the rpm package you installed in step 2. Now run a `sudo yum update`, this could take some time.
```

Comment:
```suggestion
4. Once downloaded, install it: ``sudo yum localinstall platform-and-version-specific-package-name.rpm``. This will be the rpm package you installed in step 2. Now run a ``sudo yum update``, this could take some time.
```